ecs-cluster/alerts/issues/13002_PRAVEGA_THRESHOLD_EXCEEDED_WARNING_issue.yaml

```
@@ -0,0 +1,9 @@
+- description: "If Pravega warning limit exceeded"
```

What pravega threshold is exceeded.

```suggestion
- description: "Pravega threshold exceeded"
```
The "warning" will already be listed in the issue in the gui.

---

Done